### DISCUSSION TITLE
Webp Support

### DISCUSSION THREAD

**Discussion by Schmogel** [scambier/obsidian-omnisearch#249] [Ideas]
Hey, So I have some *.webp files that I'd like to index but omnisearch doesn't recognize it as an image unless I change the file type to jpg (no conversion, just renaming). Is there a reason why omnisearch doesn't treat files ending with *.webp as valid images?

Thanks! :)

**Comment by scambier** [comment]
No particular reason, I didn't know obsidian supported webp ¯\\_(ツ)_/¯

**Reply by Schmogel** [reply]
Awesome, that was a quick fix =) thanks
